MVCCore.Mediator.Handler/ValidatorHandler/LoginValidatorHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FluentValidation;
+using FluentValidation.AspNetCore;
+using MediatR;
+using MVCCore.Mediator.Request;
+using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace MVCCore.Mediator.Handler.ValidatorHandler
+{
+    public class LoginValidatorHandler : AbstractValidator<LoginRequest>
+    {
+         
+        public LoginValidatorHandler()
+        {
+
+            SetRules();
+        }
+
+        void SetRules()
+        {
+            RuleFor(x => x.UserName)
+               .NotEmpty()
+               .WithMessage("Please enter username");
```

Comment:
Don't have to put with the message, fluent validation will take care of itself for the parent class.

Reply:
ok , I will remove